src/view/components/extensionViewForm.jsx

```diff
@@ -7,17 +7,21 @@ import ExtensionViewContext from "./extensionViewContext";
 const ExtensionViewForm = ({
   initialValues,
   getSettings,
-  validate,
-  validationSchema,
+  validateFormikState,
+  formikStateValidationSchema,
+  validateNonFormikState,
   render
 }) => {
   const formikProps = useFormik({
     initialValues,
     enableReinitialize: true,
     onSubmit: () => {},
-    validate: wrapValidateWithErrorLogging(validate),
-    validationSchema,
-    validateOnChange: false
+    validate: validateFormikState
+      ? wrapValidateWithErrorLogging(values => {
+          return validateFormikState({ values });
+        })
+      : undefined,
```

Comment:
I'm still deciding on whether I'm hipster enough to use the `validateFormikState && wrapValidateWithErrorLogging(...)` pattern instead of the ternary.